assets/app/view/game/actionable.rb

```diff
@@ -26,14 +26,15 @@ def save_user_settings(settings)
       def process_action(action)
         hotseat = @game_data[:mode] == :hotseat
         participant = @game.players.map(&:name).include?(@user&.dig('name'))
+        owner = @game.owner == @user&.dig('name')
```

Comment:
maybe just set particiant = this ? is participant gonna be different from owner?

Reply:
Right. I did it like this in the beginning, because there's a semantic distinction that I thought was going to be relevant, but looking back, it isn't.